src/app/item-popup/ItemStat.tsx

```diff
@@ -129,22 +129,41 @@ export function D1QualitySummaryStat({ item }: { item: D1Item }) {
  * it is not being modified.
  */
 function getModdedStatValue(item: DimItem, stat: DimStat) {
-  const modSockets =
-    (item.isDestiny2() &&
-      item.sockets &&
-      item.sockets.sockets.filter((socket) => {
-        const categories = idx(socket, (socket) => socket.plug.plugItem.itemCategoryHashes) || [];
-        return (
-          // these are the item category hashes for weapon mods and armor mods respectively
-          (categories.includes(1052191496) || categories.includes(4062965806)) &&
-          // we only care about the ones that modify this stat
-          Object.keys(idx(socket, (socket) => socket.plug.stats) || {}).includes(
-            String(stat.statHash)
-          )
-        );
+  if (!item.isDestiny2() || !item.sockets) {
+    return 0;
+  }
+
+  const reusableSocketCategory = item.sockets.categories.find((category) => {
+    return category.category.categoryStyle === 1;
```

Comment:
You can use the enum constant `DestinySocketCategoryStyle.Reusable` instead of `1`.